src/NuGet.Clients/NuGet.Tools/PkgCmdID.cs

```diff
@@ -15,5 +15,7 @@ internal static class PkgCmdIDList
         public const int cmdIdGeneralSettings = 0x0300;
         public const int cmdIdVisualizer = 0x0310;
         public const int cmdidDebugConsole = 0x0900;
+        public const int cmdidUpdatePackage = 0x0500;
+        public const int cmdidUpdatePackages = 0x0600;
```

Comment:
Other constants are in numeric order.

Reply:
There is a precedence for the others already being out of order actually. Take a look at a few others from this file:
```
        public const int cmdidRestorePackages = 0x300;
        public const int cmdidUpgradeNuGetProject = 0x400;
        public const int cmdidUpgradePackagesConfig = 0x410;
        public const int cmdidSourceSettings = 0x0200;
```

I looked at other PkgCmdIDList's across the solution and they also have some things in order and some not. What I've done here is grouped the last two items together, as they are related to each other, but made those in order with respect to each other.